src/BuildScriptGenerator/PlatformInstallerBase.cs

```diff
@@ -54,14 +55,19 @@ public abstract class PlatformInstallerBase
                 .AppendLine($"cd {versionDirInTemp}")
                 .AppendLine("PLATFORM_BINARY_DOWNLOAD_START=$SECONDS")
                 .AppendLine($"platformName=\"{platformName}\"")
-                .AppendLine($"if [[ \"$platformName\" = \"php\" || \"$platformName\" = \"php-composer\" ]] && [[ \"$DEBIAN_FLAVOR\" != \"stretch\" ]]; then")
+
+                // TODO: PR2 remove this if statement, as bullseye images will be supported
+                .AppendLine($"if [ \"$DEBIAN_FLAVOR\" = \"bullseye\" ]; then")
+                .AppendLine($"SDK_DEBIAN_TYPE=\"buster\"")
```

Comment:
Should this be `bullseye` or should the conditional above be checking for `buster`?

Reply:
Also do we want to use the constants generated in the `OsTypes` class for `buster` and `bullseye` strings in this file like we do below for `OsTypes.DebianStretch`?

Reply:
@cormacpayne  This was added when I saw Stella's changes in this PR: https://github.com/microsoft/Oryx/pull/1483/files 

These changes are here because we don't have any bullseye sdks, so bullseye images should pull buster SDKs until this PR adds bullseye sdks to our storage account